go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/franela/goreq v0.0.0-20170418064916-b5b0f5eb2d16
 	github.com/go-gomail/gomail v0.0.0-20150902115704-41f357289737
 	github.com/go-sql-driver/mysql v1.4.1 // indirect
+	github.com/golang-migrate/migrate v3.5.4+incompatible // indirect
```

Comment:
This is dependency of the migrate program. Since the dependency of migrate program and the server programs are too different, we moved the migrate program into folder `migrate` and has it own go.mod in that folder. Please remove it here, and run `migrate.go` in the `migrate` folder.